frontend/src/pages/admin/AdminPage.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect } from 'react';
+import useAdminStore from '../../store/admin.store';
+import style from './admin.module.css';
+import CountDisplay from '../../components/adminDetails/countDisplay/CountDisplay';
+import JobApproval from '../../components/adminDetails/jobApproval/JobApproval';
+
+function AdminPage() {
+  const jobsRequest = useAdminStore((state) => state.jobsRequest);
+  const getJobRequest = useAdminStore((state) => state.getJobRequest);
+  const usersCount = useAdminStore((state) => state.usersCount);
+  const getUserCount = useAdminStore((state) => state.getUserCount);
+  const companiesCount = useAdminStore((state) => state.companiesCount);
+  const getComaniesCount = useAdminStore((state) => state.getComaniesCount);
```

Comment:
Are we going to use those values also outside of the AdminPage? If we wont we can simply call the fetch methods here and store it in the `useState` hook

Reply:
this is not crucial, just a global store as ZUSTAND should be used more for values that are needed across multiple screens and features (e.g theme, user data... )

Reply:
Honestly, we have been asked to use Zustand. I couldn't find another purpose for it.
I might use it in another pages in when we advance a little also thought its nice new technology for us to practice :grimacing: